databox/indexer/src/handlers/phraseanet/CPhraseanetMetadata.ts

```diff
@@ -0,0 +1,25 @@
+import {PhraseanetMetaStruct, PhraseanetMetadata} from "./types";
+
+export class CPhraseanetMetadata {
```

Comment:
since the name PhraseanetMetadata (type) is already used, this prefixes **C**lasses (old microsoft best practice, now deprecated but still used) ; 
We could use aliases (not great for readability) or postfix like `PhraseanetMetadataType`. 
- name your choice...

Reply:
OK, I think the problem is the usage of a class where I think you don't need to.
In the JS manner I would have written your methods as exported functions:

```js
export function createMetaFromString(str: string): PhraseanetMetadata {
    return {
         values: [str]
    }
}
```